### DISCUSSION TITLE
How do I increase the width of the completion popup box?

### DISCUSSION THREAD

**Discussion by sferg8** [VonHeikemen/lsp-zero.nvim#371] [Q&A]
How do I increase the width of the completion popup box? I have been poking around for settings, I am new to nvim so I'm not entirely sure where to look for this.

**Comment by VonHeikemen** [comment]
You mean the "popup" that shows the documentation?

**Comment by sferg8** [comment]
Yes, sorry the documentation popup. (sorry super new to this)

**Reply by VonHeikemen** [reply]
For this you want to use the module `cmp`, and configure the `window.documentation` property. Something like this.

```lua
local cmp = require('cmp')

cmp.setup({
  window = {
    documentation = {
      max_height = 15,
      max_width = 50,
    }
  },
})
```

**Comment by sferg8** [comment]
I had a few man pages wrapping figured id see if its an easy fix
